index.js

```diff
@@ -6,4 +6,6 @@ import ActionSheetAndroid from './ActionSheetAndroid';
 
 var ActionSheet = Platform.OS === 'ios' ? ActionSheetIOS : ActionSheetAndroid;
 
-export default ActionSheet;
+//export default ActionSheet;
+
+module.exports = ActionSheet;
```

Comment:
I do that because it does not work for me. That's strange. So I change to ES5 syntax. I'm using react-native 0.29.2.